### DISCUSSION TITLE
When using hydrogen, do we have to load components from hydrogen or hydrogen-ui?

### DISCUSSION THREAD

**Discussion by bitforcesrl** [Shopify/hydrogen-ui#95] [Help] [answered]
According to this graph found in the doc [here](https://shopify.dev/custom-storefronts/hydrogen/alternate-frameworks) 
It seems that we should not import components from hydrogen-ui but use those exposed from hydrogen itself.
We've already done this question to you @frehner  as maintainer and you said that we should use hydrogen-ui.
But today we were looking to the update and we found this graph that seems to say, use hydrogen components.
Moreover not all components present in hydrogen are present in hdyrogen-ui.

What should we do? Many thanks

<img width="825" alt="Screenshot 2022-12-14 at 12 16 37" src="https://user-images.githubusercontent.com/46250701/207582147-f9edda2c-8bc5-43db-9913-70320eb7bb4f.png">

**Comment by frehner** [comment, marked as answer]
Good question! 

You should be importing from hydrogen-ui. The components in hydrogen are in maintenance mode, and will be completely removed in v2.

**Reply by bitforcesrl** [reply]
ok so maybe the graph should be updated accordingly.
we'll wait the V2 on Feb to check the new release based on remix, we imagine that we'll see hydro-ui in the package.json then

thanks for the response
